cm-vault/vault-install/main.tf

```diff
@@ -0,0 +1,39 @@
+terraform {
+  required_providers {
+    helm = {
+      source  = "hashicorp/helm"
+      version = "~> 2.5.1"
+    }
+  }
+}
+
+provider "helm" {
+  kubernetes {
+    config_path = var.kubeconfig_path
+  }
+}
+
+# deploy vault
+resource "helm_release" "vault" {
+  name       = "vault"
+  repository = "https://helm.releases.hashicorp.com"
+  chart      = "vault"
+  namespace        = "vault"
+  create_namespace = true
+
+  set {
+    # This installs a single Vault server with a memory storage backend.
```

Comment:
Suggestion: `# This installs a single unsealed Vault server in the insecure dev mode with a memory storage backend.`